python/ray/tests/test_ray_init.py

```diff
@@ -123,6 +125,26 @@ def test_ports_assignment(ray_start_cluster):
             **pre_selected_ports, min_worker_port=25000, max_worker_port=35000)
 
 
+@pytest.mark.skipif(sys.platform != "linux", reason="skip except linux")
```

Comment:
this test only works on linux, so skip for other OS.

Reply:
Why is that?

Reply:
This UT wants to simulate two nodes(with different node_ip_address). So we use the loopback address. However, we could not use a loopback address except 127.0.0.1 on macOS by default. https://superuser.com/questions/458875/how-do-you-get-loopback-addresses-other-than-127-0-0-1-to-work-on-os-x